tests/Validation/ValidationAddFailureTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Illuminate\Tests\Validation;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Validation\Validator;
+
+class ValidationAddFailureTest extends TestCase
+{
+    /**
+     * Making Validator using ValidationValidatorTest.
+     *
+     * @return Validator
+     */
+    public function makeValidator(): Validator
```

Comment:
Laravel does not use return types in the code like this.

Reply:
Laravel 5.6 Server Requirements [PHP >= 7.1.3](https://laravel.com/docs/5.6), meaning it is legal, 
 you can even use nullable types there if needed. Plus, it is a test case. It is the best place for introducing innovations.

Reply:
Are you sure you want me to remove the returning type declaration?

Reply:
No other method in the framework uses return type hints. It would be weird to have a single method using them.

Pretty much everything has docblock declarations, so it's not like we don't know what's being returned.

Reply:
Laravel's code style conventions forbid using php 7 style return types.

Reply:
Done, return type declaration removed. P.S. It is sad that no method in the framework uses return type hints. :(

Reply:
@GrahamCampbell ,  Thank you for explanation. The requested change is done.

Reply:
@GrahamCampbell , I still can see a big red button with a cross from your change request. Does it mean you have not accept my change? I really don't know.